cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -167,7 +167,6 @@ systemd:
       --cni-conf-dir=/etc/kubernetes/cni/net.d \
       --network-plugin=cni \
       --container-runtime=docker \
-      --cadvisor-port=4194 \
```

Comment:
on prupose?

Reply:
Yes, this flag is removed in 1.12 and replaced by the cadvisor-daemonset in our setup.